### ISSUE TITLE
Custom reporting possibility

### ISSUE THREAD

**Issue by mattiasvdm** [munkireport/munkireport-php#952]
Feature request:
Pretty often we have to report on stuff which isn't in munkireport. The last example was a specific build version for Safari since Apple released version 11.0.2 two times. Therefore we currently create a script which outputs in the "Computerinfo" fields. However, this is limited to only four fields. For me it would be useful to have a blank sheet with a fixed (or variable) number of fields which can be filled with anything. A payload free package with a postinstall script could output in a dedicated plist file which would be uploaded by the postflight process.
The upside of this is that you can report on virtually anything. The downside could be that this leads to less feature requests.
One suggestion of how the sheet could look like:
Name | Serial Number | Username | Check-in | Customfield#1 | Customfield#2 | etc..

**Comment by tuxudo** [comment]
Isn't this what the munkifacts module does? I don't think it's still only in the WIP branch.

**Comment by mattiasvdm** [comment]
Yes, this actually looks a lot like it. I see it's in the latest RC. I'll play around with it.

**Comment by tuxudo** [comment]
Issue resolved, also look into the https://github.com/tuxudo/extension_attributes module
